system-addon/content-src/lib/section-menu-options.js

```diff
@@ -0,0 +1,46 @@
+import {actionCreators as ac, actionTypes as at} from "common/Actions.jsm";
+
+/**
+ * List of functions that return items that can be included as menu options in a
+ * SectionMenu. All functions take the section as the only parameter.
+ */
+export const SectionMenuOptions = {
+  Separator: () => ({type: "separator"}),
+  RemoveSection: section => ({
+    id: "section_menu_action_remove_section",
+    icon: "dismiss",
+    action: ac.SetPref(section.showPrefName, false),
+    userEvent: "SECTION_MENU_REMOVE"
+  }),
+  CollapseSection: section => ({
+    id: "section_menu_action_collapse_section",
+    icon: "collapse",
+    action: ac.SetPref(section.collapsePrefName, true),
+    userEvent: "SECTION_MENU_COLLAPSE"
+  }),
+  ExpandSection: section => ({
+    id: "section_menu_action_expand_section",
+    icon: "collapse",
+    action: ac.SetPref(section.collapsePrefName, false),
+    userEvent: "SECTION_MENU_EXPAND"
+  }),
+  ManageSection: section => ({
+    id: "section_menu_action_manage_section",
+    icon: "settings",
+    action: {type: at.SETTINGS_OPEN},
+    userEvent: "SECTION_MENU_MANAGE"
+  }),
+  AddTopSite: section => ({
+    id: "section_menu_action_add_topsite",
+    icon: "add",
+    action: {type: at.TOP_SITES_EDIT, data: {index: -1}},
+    userEvent: "SECTION_MENU_ADD_TOPSITE"
+  }),
+  PrivacyNotice: section => ({
+    id: "section_menu_action_privacy_notice",
+    icon: "privacy",
+    action: {type: at.OPEN_LINK, data: {url: section.privacyNoticeURL}},
```

Comment:
This wants an `OnlyToMain` (right now it sends only to itself/content, so it does nothing.)

Reply:
oops. guess I didn't test that 🤦‍♂️